src/construction.cpp

```diff
@@ -1520,8 +1520,12 @@ void finalize_constructions()
         }
     }
 
+    // previously dead code became live in #17451 and caused #17443 and #17449
+    // is temporarily disabled pending refactor as construction_id = string_id<construction>
+    /*
```

Comment:
We typically remove unused code instead of commenting it out (because you always able to restore it using git history).